chromadb/api/types.py

```diff
@@ -337,9 +337,21 @@ def validate_embeddings(embeddings: Embeddings) -> Embeddings:
         raise ValueError(
             f"Expected each embedding in the embeddings to be a list, got {embeddings}"
         )
+    _new_embeddings: Embeddings = list()
     for embedding in embeddings:
         if not all([isinstance(value, (int, float)) for value in embedding]):
-            raise ValueError(
-                f"Expected each value in the embedding to be a int or float, got {embeddings}"
-            )
-    return embeddings
+            try:
```

Comment:
I think rather than generic casting we should just explicitly support numpy at the level o Collections.py

For numpy we can check if its an np array and then explicitly call .tolist()

Reply:
@HammadB, take a look now

Reply:
@tazarov lets make the change I mentioned above